scripts/external.test

```diff
@@ -15,26 +15,35 @@ if ! ./examples/client/client -V | grep -q 3; then
     exit 77
 fi
 
+# These tests do not run by default, we will call these opt in. There are
+# additional default tests that require network connection, those are
+# opt out.
+# Not defining WOLFSSL_EXTERNAL_TEST runs only opt out.
+# Defining WOLFSSL_EXTERNAL_TEST!=0 runs opt out and opt in.
+# Defining WOLFSSL_EXTERNAL_TEST=0  does not run opt in or opt out.
```

Comment:
I think the code speaks for itself, particularly since it `echo`s explanations that double as documentation.

I.e. this comment block should be snipped out.